awx/main/utils/ansible.py

```diff
@@ -44,7 +45,12 @@ def could_be_playbook(project_path, dir_path, filename):
     # show up.
     matched = False
     try:
-        for n, line in enumerate(open(playbook_path)):
+        for n, line in enumerate(codecs.open(
+            playbook_path,
+            'r',
+            encoding='utf-8',
+            errors='ignore'
```

Comment:
Just ignore anything we can't decode; the purpose of this code is just to look for certain regular expression matches.